pipelines/backup/cassandra-backup/Jenkinsfile

```diff
@@ -31,8 +31,20 @@ node() {
                 ansible_playbook_run(values)
             }
         }
+        currentBuild.result = 'SUCCESS'
     }
     catch (err) {
+        currentBuild.result = 'FAILURE'
+        slackSend (
+            channel: "${env.NOTIFY_SLACK_CHANNEL}",
```

Comment:
@santosh0705 This would send slack notifications of all env's to same channel? Do you feel we need to have separate channels per env or all in once?
Slack alerts require a integration key. Do we have this already?
Also this is a global variable which isn't currently defined, please provide the value for this.